app/pages/applicantportal/index.tsx

```diff
@@ -89,9 +93,11 @@ const Home = ({
         </>
       );
 
-    const formattedClosingDate = DateTime.fromISO(openIntake.closeTimestamp)
-      .minus({ minutes: 30 })
-      .toLocaleString(DateTime.DATETIME_FULL);
+    // not ideal but checking type here for now due mockOpenIntake applicantportal/index test mocking useFeature and returning object
+    const formattedClosingDate = dateTimeSubtracted(
+      openIntake.closeTimestamp,
+      typeof showSubtractedTime === 'number' ? showSubtractedTime : 0
```

Comment:
Just flagging this here, I had to do a type check to get an unrelated test in `applicantportal/index.test.ts` that failed due to mocking `useFeature` with an unrelated feature that returned an object. Spent a bit of time trying to get it to pass without this so I'm open to suggestions, though in the absence of time it was easiest to do this.